test/e2e/suites.go

```diff
@@ -35,6 +35,7 @@ func AfterSuiteActions(ctx context.Context) {
 	if framework.TestContext.ReportDir != "" {
 		framework.CoreDump(framework.TestContext.ReportDir)
 	}
+	// TODO: nothing is using this? what should we do with it?
```

Comment:
Deprecate the command line option for it in 1.35, then remove it in 1.36?

Reply:
Or start using it, at least in some jobs? We recently had a case where it would have been nice to get more insights into control plane load during testing: https://github.com/kubernetes/kubernetes/issues/133531#issuecomment-3221083935

cc @tkashem

Reply:
We would need to figure out how what to do with the data, though. A simple howto somewhere would be useful.